assets/src/components/notificationBellIcon.tsx

```diff
@@ -17,8 +17,8 @@ const NotificationBellIcon = ({
   )
   const unreadBadge: boolean = unreadNotifications.length > 0
 
-  return notificationBellIcon(
-    className([
+  return icon({
+    className: className([
```

Comment:
**question:** is there a reason you can't just do `<NotificationBellIcon className={className(...)} />` here?

Reply:
Ah, right, understood on point one. I guess on point two I find the JSX syntax easier to read in this case (see other comment thread) but I don't feel super strongly about it.